pytest.ini

```diff
@@ -1,2 +1,134 @@
 [pytest]
 norecursedirs = .git build ci dist extern release tools unit venv
+pep8ignore =
+    * E111 E114 E115 E116 E121 E122 E123 E124 E125 E126 E127 E128 E129 E131 E226 E240 E241 E242 E243 E244 E245 E246 E247 E248 E249 E265 E266 E704 W503
```

Comment:
We could enable them again one-by-one? Maybe? *shrugs*

Could replace `*` with the files we don't expect to pass.